packages/experiments/index.html

```diff
@@ -1,62 +0,0 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0" user-scalable="no" />
-    <meta property="og:title" content="WordTest with a really long title that will wrap around to the second line but not the third line" />
```

Comment:
Just noticed this weird bit in the experiments index.html...not sure what this was doing and whether I should bring it back.

Reply:
hmmm, viewport for mobile stuff